editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -1109,4 +1160,44 @@ describe('Double click on resize edge', () => {
     expect(div.style.width).toEqual('637px')
     expect(div.style.height).toEqual(minContent)
   })
+
+  it("not applicable when children don't participate in the layout", async () => {
+    const editor = await renderTestEditorWithCode(
+      projectForEdgeDblClickWithPosition('absolute', 'absolute'),
+      'await-first-dom-report',
+    )
+    const div = await doDblClickTest(editor, edgeResizeControlTestId(EdgePositionBottom))
+    expect(div.style.width).toEqual('637px')
+    expect(div.style.height).toEqual('445px')
+  })
+
+  it('not applicable when children are positioned absolute', async () => {
+    const editor = await renderTestEditorWithCode(
+      projectForEdgeDblClickWithPosition('absolute', 'absolute'),
+      'await-first-dom-report',
+    )
+    const div = await doDblClickTest(editor, edgeResizeControlTestId(EdgePositionBottom))
+    expect(div.style.width).toEqual('637px')
+    expect(div.style.height).toEqual('445px')
+  })
```

Comment:
It doesn't seem like there is much point in having both of these tests as they're doing the same thing. Maybe it would be better to have one of these, and a test without any children?

Reply:
absolutely, fixing it

Reply:
https://github.com/concrete-utopia/utopia/pull/3071/commits/00192c37ce0a457f3a4f0a1f33947fa11d94f912